src/js/scrollArea.jsx

```diff
@@ -148,6 +163,11 @@ export default class ScrollArea extends React.Component{
         this.setState(newState);
     }
 
+    convertLinesToPixels(lines) {
+        var pixels = lineHeight(findDOMNode(this.refs.content));
```

Comment:
Ok now I understand. Yes, I agree. I think it's a good idea to compute it once instead of each time. I will have a look into this.